src/Species.cpp

```diff
@@ -193,7 +193,7 @@ namespace NEAT
     // returns a completely random individual
     Genome Species::GetRandomIndividual(RNG &a_RNG) const
     {
-        if (BOOST_UNLIKELY(m_Individuals.empty())) // no members yet, return representative
+        if (m_Individuals.empty()) // no members yet, return representative
```

Comment:
if we want, there is an `unlikely` in gcc at least

Reply:
It is not standard. Also I think this really is not the performance bottleneck in this codebase. To me it feels it was here because the original author thought it was fun to use :)